src/Http/Http.Results/src/Results.cs

```diff
@@ -0,0 +1,1243 @@
+        public static IResult Created(Uri uri, object? value)
+        {
+            if (uri == null)
+            {
+                throw new ArgumentNullException(nameof(uri));
+            }
+
+            return new CreatedResult(uri, value);
+        }
+
+        /// <summary>
+        /// Creates a <see cref="CreatedAtRouteResult"/> object that produces a <see cref="StatusCodes.Status201Created"/> response.
+        /// </summary>
+        /// <param name="routeName">The name of the route to use for generating the URL.</param>
+        /// <param name="value">The content value to format in the entity body.</param>
+        /// <returns>The created <see cref="CreatedAtRouteResult"/> for the response.</returns>
+        public static IResult CreatedAtRoute(string? routeName, object? value)
+            => CreatedAtRoute(routeName, routeValues: null, value: value);
+
+        /// <summary>
+        /// Creates a <see cref="CreatedAtRouteResult"/> object that produces a <see cref="StatusCodes.Status201Created"/> response.
+        /// </summary>
+        /// <param name="routeValues">The route data to use for generating the URL.</param>
+        /// <param name="value">The content value to format in the entity body.</param>
+        /// <returns>The created <see cref="CreatedAtRouteResult"/> for the response.</returns>
+        public static IResult CreatedAtRoute(object? routeValues, object? value)
+            => CreatedAtRoute(routeName: null, routeValues: routeValues, value: value);
+
+        /// <summary>
+        /// Creates a <see cref="CreatedAtRouteResult"/> object that produces a <see cref="StatusCodes.Status201Created"/> response.
+        /// </summary>
+        /// <param name="routeName">The name of the route to use for generating the URL.</param>
+        /// <param name="routeValues">The route data to use for generating the URL.</param>
+        /// <param name="value">The content value to format in the entity body.</param>
+        /// <returns>The created <see cref="CreatedAtRouteResult"/> for the response.</returns>
+        public static IResult CreatedAtRoute(string? routeName, object? routeValues, object? value)
+            => new CreatedAtRouteResult(routeName, routeValues, value);
+
+        #endregion
+
+        #region AcceptedResult
+        /// <summary>
+        /// Creates a <see cref="AcceptedResult"/> object that produces an <see cref="StatusCodes.Status202Accepted"/> response.
+        /// </summary>
+        /// <returns>The created <see cref="AcceptedResult"/> for the response.</returns>
+        public static IResult Accepted()
+            => new AcceptedResult();
+
+        /// <summary>
+        /// Creates a <see cref="AcceptedResult"/> object that produces an <see cref="StatusCodes.Status202Accepted"/> response.
+        /// </summary>
+        /// <param name="value">The optional content value to format in the entity body; may be null.</param>
```

Comment:
Might 'response body' be clearer language than 'entity body'?

Reply:
Sure. I think this comment might have been from WebAPI2 which tends to use the [RFC names](https://www.w3.org/Protocols/rfc2616/rfc2616-sec7.html)